ckanext/harvest/logic/action/get.py

```diff
@@ -310,6 +312,38 @@ def harvesters_info_show(context,data_dict):
 
     return available_harvesters
 
+@side_effect_free
+def harvest_log_list(context,data_dict):
+    '''Returns a list of harvester log entries grouped by level.
+
+    :param per_page: number of logs to be shown default: 100
+    :param offset: use with ``per_page`` default: 0
+    :param level: filter log entries by level(debug, info, warning, error, critical)
+    '''
+
+    check_access('harvest_log_list', context, data_dict)
+
+    model = context['model']
+    session = context['session']
+
+    per_page = data_dict.get('per_page', 100)
```

Comment:
This needs an enforced cap - ex: `min(int(data_dict.get('per_page', 100)), 100)`